var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -77,6 +78,7 @@ class PyMdanalysis(PythonPackage):
     depends_on("py-parmed", when="+extra_formats @2.6.0:", type=("build", "run"))
```

Comment:
The "can't be concretized with py-numpy" bug was fixed, so I think you can uncomment gsd now if you want.

Reply:
Thanks for the info. I added GSD versions `3.x`. `py-chemfiles` seemed to have issues on macOS with `py-cmake@3.22`, so I added a few more recent versions and confirmed that

```bash
spack install py-mdanalysis@2.7.0 +extra_formats +analysis
```

works correctly.